concourse/pipelines/windows-image-build.yaml

```diff
@@ -3829,6 +3907,8 @@ jobs:
     file: publish-version/version # produced from generate-version task
   - task: publish-2019-for-containers
     file: guest-test-infra/concourse/tasks/daisy-publish-images.yaml
+    params:
+      ROLLOUT_RATE: 0
```

Comment:
i see you're setting this to 0 in many places, should the default be zero?

Reply:
0 is desired value for those “publish to testing” workflows. The other publish jobs (to staging and prod) should still uses 60.